rules/windows/discovery_posh_invoke_sharefinder.toml

```
@@ -17,7 +17,47 @@ index = ["winlogbeat-*", "logs-windows.*"]
 language = "kuery"
 license = "Elastic License v2"
 name = "PowerShell Share Enumeration Script"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating PowerShell Share Enumeration Script
+
+PowerShell is one of the main tools system administrators use for automation, report routines, and other tasks. This
+makes it available for use in various environments, and creates an attractive way for attackers to execute code.
+
+Attackers can use PowerShell to enumerate shares looking for sensitive data like documents, scripts, and other kinds of
+valuable data for encryption, exfiltration, and lateral movement.
```

This sounds like the shares are looking for data, not the attackers. Maybe this:
```suggestion
Attackers can use PowerShell to enumerate shares to search for sensitive data like documents, scripts, and other kinds of
valuable data for encryption, exfiltration, and lateral movement.
```